docs/data-cookbooks/browse-generation.md

```diff
@@ -155,7 +155,7 @@ This will cause kes to export the workflows in the new file along with the other
 Ensure the following lambdas are in your deployment's lambdas.yml (reference the [example lambdas.yml](https://github.com/nasa/cumulus/blob/master/example/lambdas.yml)):
 
 ```
-DiscoverGranulesNoVpc:
+DiscoverGranules:
```

Comment:
should this object and the following object in the `SyncGranule` change have `launchInVpc: true` set?

Reply:
yeah I think you're probably right here.

Reply:
I updated it, thanks!